react/SlideToggle/SlideToggle.less

```diff
@@ -50,7 +51,8 @@
 
 .label {
   padding: 0;
-  display: inline;
+  line-height: @grid-row-height * 3;
+  transform: none;
```

Comment:
I was chatting with @mattsjones yesterday about how there are some instances where there's a need for responsive text without baseline correction and that perhaps we should make that a thing in styleguide.  I don't think this is something we should use or need often but there are some cases where it is useful, like the slide toggle.  @markdalgleish, do you have any thoughts on this given you've just done a lot of work on responsive text in the styleguide?